ome_zarr/dask_utils.py

```diff
@@ -8,23 +8,27 @@
 # See https://github.com/toloudis/ome-zarr-py/pull/1
 
 
-def resize(image: da.Array, output_shape: Tuple[int, ...], *args, **kwargs) -> da.Array:
-    """
+def resize(
+    image: da.Array, output_shape: Tuple[int, ...], *args: int, **kwargs: str
```

Comment:
I got stuck staring at this args, kwargs type hint stuff last night...  it's not great.  The implication here seems to be  that all args types are ints and all kwargs types are strings, but if it works it works.

Reply:
Agreed. Do we know anything about what we want those arguments to be, otherwise I'd suggest:
```
*args: Any, **kwargs: Any
```

Reply:
I think we go for `Any` since the types of extra resize args (see https://scikit-image.org/docs/dev/api/skimage.transform.html#resize) are various

Reply:
Pushed that change to use `Any`